enumeration/extension/gx-name-hidden.yaml

```diff
@@ -0,0 +1,17 @@
+%YAML 1.2
+---
+lang: en-US
+
+type: enumeration
+
+uri: https://tychonievich.github.io/gx-g7-names#hidden
+
+extension tags:
+  - _NPQ_HIDDEN
```

Comment:
So this is not part of GEDCOM-X? (not `_GX_HIDDEN`)
If so, then perhaps the PR title isn't quite right.